clusterctl/examples/openstack/cluster.yaml.template

```diff
@@ -0,0 +1,16 @@
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: Cluster
+metadata:
+  name: test1
+spec:
+    clusterNetwork:
+        services:
+            cidrBlocks: ["10.96.0.0/12"]
```

Comment:
what is this CIDR? how does one figure this out for their openstack environment?

Reply:
This CIDR is for k8s allocate clusterIP for it's service, not relay on openstack environment.